tests/test_optics.py

```diff
@@ -1,5 +1,8 @@
+import numpy as np
 import pytest
 
+import glidertools as gt
```

Comment:
Could you import each function individually like this:
```python
from glidertools.optics import backscatter_dark_count
```
This will immediately raise import problems at the beginning, which makes debugging easier down the line.